ci/test/02_run_container.sh

```diff
@@ -44,18 +44,21 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
   CI_DEPENDS_MOUNT="type=volume,src=${CONTAINER_NAME}_depends,dst=$DEPENDS_DIR/built"
   CI_DEPENDS_SOURCES_MOUNT="type=volume,src=${CONTAINER_NAME}_depends_sources,dst=$DEPENDS_DIR/sources"
   CI_PREVIOUS_RELEASES_MOUNT="type=volume,src=${CONTAINER_NAME}_previous_releases,dst=$PREVIOUS_RELEASES_DIR"
+  CI_BUILD_MOUNT=""
```

Comment:
feel-free-to-disregard-followup: After thinking more about my comment below, (https://github.com/bitcoin/bitcoin/pull/31428/files#r1897104374) It is because we might not use *any* volume mount for the build dir, and we want to be able to leave it blank here.

But I'm wondering why the build directory doesn't get it's own volume, why doesn't the same reasoning from #27033 apply here?